app/middlewares/param-validation.js

```diff
@@ -0,0 +1,12 @@
+const { conflictError } = require('../errors');
+
+const paramValidation = param => (req, res, next) => {
+  const queryParam = req.query[param];
+  if (!queryParam) {
+    return next(conflictError('param schema', `The ${param} parameter is required.`));
+  }
+
+  return next();
+};
```

Comment:
You can check this with the schema validator

Reply:
Done